videoio compilation error

##### System information (version)
- OpenCV => 3.1.0-dev, master branch
- Operating System / Platform => Windows 64 bits
- Compiler => mingw32 64 bits

##### Detailed description
Compilation error in when building the videoio module:
```In file included from D:/opencv/modules/core/include/opencv2/core.hpp:53:0,
                 from D:/opencv/modules/videoio/include/opencv2/videoio.hpp:46,
                 from D:\opencv\modules\videoio\src\precomp.hpp:45,
                 from D:\opencv\modules\videoio\src\cap_ffmpeg.cpp:42:
D:\opencv\modules\videoio\src\cap_ffmpeg.cpp: In constructor 'icvInitFFMPEG::icvInitFFMPEG()':
D:/opencv/modules/core/include/opencv2/core/version.hpp:61:30: error: expected ',' or ';' before 'L'
 #define CVAUX_STRW_EXP(__A)  L#__A
                              ^
D:/opencv/modules/core/include/opencv2/core/version.hpp:62:30: note: in expansion of macro 'CVAUX_STRW_EXP'
 #define CVAUX_STRW(__A)      CVAUX_STRW_EXP(__A)
                              ^
D:\opencv\modules\videoio\src\cap_ffmpeg.cpp:112:13: note: in expansion of macro 'CVAUX_STRW'
             CVAUX_STRW(CV_MAJOR_VERSION) CVAUX_STRW(CV_MINOR_VERSION) CVAUX_STRW(CV_SUBMINOR_VERSION)
             ^
modules\videoio\CMakeFiles\opencv_videoio.dir\build.make:237: recipe for target 'modules/videoio/CMakeFiles/opencv_videoio.dir/src/cap_ffmpeg.cpp.obj' failed
mingw32-make[2]: *** [modules/videoio/CMakeFiles/opencv_videoio.dir/src/cap_ffmpeg.cpp.obj] Error 1
CMakeFiles\Makefile2:6768: recipe for target 'modules/videoio/CMakeFiles/opencv_videoio.dir/all' failed
mingw32-make[1]: *** [modules/videoio/CMakeFiles/opencv_videoio.dir/all] Error 2
Makefile:159: recipe for target 'all' failed
mingw32-make: *** [all] Error 2```

##### Steps to reproduce
This problem (in _version.hpp_) is visible since:
> Revision: 1f26e7347841fa9c65729a4432cff4bb1e7d184d
> Author: Alexander Alekhin <alexander.alekhin@itseez.com>
> Date: 08/07/2016 14:47:50
> Message:
> ffmpeg: try to load ffmpeg wrapper dll from the current module directory
> Modified: modules/videoio/src/cap_ffmpeg.cpp

A PR will follow.

--- a/modules/core/include/opencv2/core/version.hpp
+++ b/modules/core/include/opencv2/core/version.hpp
@@ -58,7 +58,7 @@
 #define CVAUX_STR_EXP(__A)  #__A
 #define CVAUX_STR(__A)      CVAUX_STR_EXP(__A)
 
-#define CVAUX_STRW_EXP(__A)  L#__A
+#define CVAUX_STRW_EXP(__A)  L ## #__A
 #define CVAUX_STRW(__A)      CVAUX_STRW_EXP(__A)
 
 #define CV_VERSION          CVAUX_STR(CV_VERSION_MAJOR) "." CVAUX_STR(CV_VERSION_MINOR) "." CVAUX_STR(CV_VERSION_REVISION) CV_VERSION_STATUS

